core/controller/src/whisk/core/controller/Backend.scala

```diff
@@ -60,17 +77,22 @@ object WhiskServices extends LoadbalancerRequest {
      * @return function that accepts an HttpRequest, posts request to load balancer
      * and returns the HTTP response from the load balancer as a future
      */
-    def postLoadBalancerRequest(config: WhiskConfig, timeout: Timeout = 10 seconds)(
-        implicit as: ActorSystem, ec: ExecutionContext): HttpRequest => Future[LoadBalancerResponse] =
-        request(config.loadbalancerHost, timeout)
+    def performLoadBalancerRequest(config: WhiskConfig, timeout: Timeout = 10 seconds)(
+        implicit as: ActorSystem, ec: ExecutionContext): (String, ActivationMessage) => Future[LoadBalancerResponse] = {
+        val requester = request(config.loadbalancerHost, timeout)
+        (component: String, messages : ActivationMessage) => {
+            val req = Post(component, messages.toJson.asJsObject)
+            requester(req)
+        }
+    }
 }
 
 trait WhiskServices {
     /** An entitlement service to check access rights. */
     protected val entitlementService: EntitlementService
 
-    /** An HTTP client to post backend requests via Whisk load balancer. */
-    protected val postLoadBalancerRequest: HttpRequest => Future[LoadBalancerResponse]
+    /** Synchronously perform a request to the load balancer.  */
```

Comment:
Premature comment - it's not synchronous as of this commit yet since this is preparatory to the final piece.